src/app/helpers/pageAnchor.test.js

```diff
@@ -0,0 +1,16 @@
+import urlWithPageAnchor from './pageAnchor';
+
+describe('urlWithPageAnchor', () => {
+  beforeEach(() => {
+    window.location.hash = '';
+  });
+
+  it('should return false when no anchor', () => {
+    expect(urlWithPageAnchor()).toBeFalsy();
```

Comment:
```suggestion
    expect(urlWithPageAnchor()).toBe(false);
```

Ideally we should be explicit that this returns the boolean false.

Reply:
Spoke IRL and this test returns an empty string `''` so is fine to be `falsey`. @rhell is there a test that checks that `urlWithPageAnchor()` returns `false` if `onClient()` is false?

Reply:
_there is now..._